test/test_utils/Mocks.h

```diff
@@ -79,21 +80,31 @@ class MockSubscriber : public Subscriber<T> {
   /**
    * Block the current thread until either onSuccess or onError is called.
    */
-  void awaitTerminalEvent() {
+  void awaitTerminalEvent(
+      std::chrono::milliseconds timeout = std::chrono::seconds(1)) {
```

Comment:
it required me to import the literals namespace. the compiler has complained in fbcode side about including standard namespaces in the header file and required me to make an unnamed namespace. I just wanted to write it like this and get rid of the "using" line.